src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -1073,10 +1101,22 @@ HeapWord* G1CollectedHeap::expand_and_allocate(size_t word_size) {
   log_debug(gc, ergo, heap)("Attempt heap expansion (allocation request failed). Allocation request: " SIZE_FORMAT "B",
                             word_size * HeapWordSize);
 
-
   if (expand(expand_bytes, _workers)) {
     _hrm.verify_optional();
     _verifier->verify_region_sets_optional();
+    return true;
+  }
+  return false;
+}
+
+// Attempting to expand the heap sufficiently
```

Comment:
```suggestion
// Attempts to expand the heap sufficiently
```
* the formatting of this comment paragraph is very "jagged"
* this looks like a comment that should go into the .hpp file